generate/generate.go

```diff
@@ -679,12 +679,12 @@ func lastCap() capability.Cap {
 	return last
 }
 
-func checkCap(c string, hostSpecific bool) error {
+func ValidCap(c string, hostSpecific bool) error {
 	isValid := false
 	cp := strings.ToUpper(c)
```

Comment:
I think we can drop this.  The spec [says nothing about case-insensitivity][1], and the values listed in the referenced man page are uppercase.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/config.md#process-configuration